### PR TITLE
create getCompanyBalance function

### PR DESCRIPTION
**Pull request by xXAndrexPereXx**
This PR was created to obtain the balance sheet of a company.

### CODE REVIEW

File: src/clients/FlexbaseClient.Company.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    minimumDue: number;
+    billDate: string;
+    graceDate: string;
+  }
+
+export class FlexbaseClientCompany extends FlexbaseClientBase {
+
+    async getCompanyBalance(companyId: string): Promise<CompanyBalance | null> {
+        try {
+            return await this.client.url(`/servicing/minimumDue/${companyId}`).get().json();
```
Comment (tophercf) [inline src/clients/FlexbaseClient.Company.ts]: hm, seen this in a few places - i think the function is marked async, so you may not need to necessarily `await` the call.  async marks the function and returning a promise, so the caller will be awaiting your instantiated method

Comment (tophercf) [reply src/clients/FlexbaseClient.Company.ts]: here's a bit of info on some performance impacts of doing await within an async function: https://eslint.org/docs/latest/rules/no-return-await

I'm not entirely sure, but i also might check the .json() call you are making - if this is the axios client, you're essentially making this a synchronous function

Comment (xXAndrexPereXx) [reply src/clients/FlexbaseClient.Company.ts]: I got it

**Review by tophercf — APPROVED** [review]

**Review by Mariana0711 — APPROVED** [review]

**Pull request merged**